### PR TITLE
Update s3 documentation

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary

Update S3 documentation to remove reference to access via S3 the IFCA endpoint, and the creation of the EC2 access keys
---

**Related issue :**

### CODE REVIEW

File: content/en/users/online-storage/object-storage/_index.md [hunk 1]
```diff
@@ -338,33 +338,15 @@ called [Simple Storage Service](https://docs.aws.amazon.com/AmazonS3/latest/user
...
-values, as those are needed in subsequent commands that use the S3 protocol.
-{{% /alert %}}
-
-To list containers/objects via the S3 protocol, use the command:
+For example, to list containers/objects via the S3 protocol, use the command:
 
 ```shell
 $ davix-ls --s3accesskey 'access' --s3secretkey 'secret' \
-  --s3alternate s3s://api.cloud.ifca.es:8080/swift/v1/test-egi
+  --s3alternate s3s://s3.cl2.du.cesnet.cz/bucket-name
```
Comment (sebastian-luna-valero) [inline content/en/users/online-storage/object-storage/_index.md:349]:
```suggestion
  --s3alternate s3s://s3.cl2.du.cesnet.cz/<bucket-name>
```

File: content/en/users/online-storage/object-storage/_index.md [hunk 2]
```diff
@@ -374,14 +356,17 @@ and delete objects from the storage.
 
 The [EGI Data Transfer](../../data-transfer) service can move files
 to and from object storages that are compatible with the S3 protocol.
-You will have to upload the EC2 access keys to the EGI Data Transfer
+You will have to upload the access keys to the EGI Data Transfer
```
Comment (sebastian-luna-valero) [inline content/en/users/online-storage/object-storage/_index.md:359]:
```suggestion
You will have to upload the `access` keys to the EGI Data Transfer
```

**Review by sebastian-luna-valero — APPROVED** [review]
Thanks!

Two minor suggestions only.

**Comment by andrea-manzi** [comment]
ok i applied the changes, but we need an admin to merge them, and Baptiste is off:-p

**Comment by andrea-manzi** [comment]
@enolfc do you have the permission for merging?

**Pull request merged**